src/components/AddForm.js

```diff
@@ -42,81 +37,35 @@ export class AddForm extends Component {
     });
   };
 
-  handleAddImage = e => {
-    if (e.target.files[0]) {
-      const image = e.target.files[0];
-      this.setState(() => ({ image }));
-    }
-    const imageName = e.target.files[0].name;
-    this.setState({ imageName });
-    console.log(imageName);
-  };
-
-  handleImgDescription = e => {
-    this.setState({
-      imgTextarea: e.target.value
-    });
-  };
-
   handleSubmit = e => {
     e.preventDefault();
-    const { image } = this.state;
-    const uploadTask = storage.ref(`images/${image.name}`).put(image);
-    uploadTask.on(
-      "state_changed",
-      snapshot => {
-        // progress function ....
-        const progress = Math.round(
-          (snapshot.bytesTransferred / snapshot.totalBytes) * 100
-        );
-        this.setState({ progress });
-      },
-      error => {
-        // error function ....
-        console.log(error);
-      },
-      () => {
-        storage
-          .ref("images")
-          .child(image.name)
-          .getDownloadURL()
-          .then(url => {
-            const plantsRef = firebase.database().ref("plants");
-            const plant = {
-              name: this.state.text,
-              description: this.state.textarea,
-              tags: {
-                tagLittleSun: this.state.tagLittleSun,
-                tagMoreSun: this.state.tagMoreSun,
-                tagLittleWater: this.state.tagLittleWater,
-                tagMoreWater: this.state.tagMoreWater,
-                tagSafe: this.state.tagSafe,
-                tagPoison: this.state.tagPoison
-              },
-              image: {
-                url: url,
-                description: this.state.imgTextarea
-              }
-            };
-            plantsRef.push(plant);
-            this.setState({
-              text: "",
-              textarea: "",
-              tagPoison: false,
-              tagSafe: false,
-              tagMoreSun: false,
-              tagMoreWater: false,
-              tagLittleSun: false,
-              tagLittleWater: false,
-              imgTextarea: "",
-              image: null,
-              progress: 0,
-              imageName: ""
-            });
-            alert("Dodano roślinkę!");
-          });
+
+    const plantsRef = firebase.database().ref("plants");
+    const plant = {
+      name: this.state.text,
+      description: this.state.textarea,
+      date: this.state.date,
```

Comment:
I recommend using Firebase server time: https://stackoverflow.com/questions/24202641/how-can-you-get-the-server-time-from-firebase

Reply:
Here is the official doc: https://firebase.google.com/docs/reference/js/firebase.database.ServerValue

Reply:
Ok, I changed it.

Reply:
You did or you will? :)

Reply:
I approve the PR upfront - nothing more from me here :)